containers/mysql/README.md

```diff
@@ -0,0 +1,14 @@
+# MySQL container
+
+This directory provides a Dockerfile to build a MySQL container for [MOCO](https://github.com/cybozu-go/moco).
+This also provides `moco-init` command to initialize MySQL data for MOCO.
```

Comment:
From this update, mysql images will not contain moco-init.

Reply:
Remove it: https://github.com/cybozu-go/moco/pull/533/commits/027acfe82595ec801aed515ffb96d87e4e938048